generation/tasks/RivetMerge.py

```diff
@@ -286,14 +309,12 @@ def run(self):
                 self.input()["analyses"]["collection"].targets.values()
             )
         )
+        # Rivet requires the so files to be named correctly, so we can't just use localize
         local_so_files = []
-        for target in self.input()["analyses"]["collection"].targets.values():
-            with target.localize("r") as so_file:
-                local_path = os.path.join(
-                    dirname, str(os.path.basename(so_file.path)).split("_", 1)[1]
-                )
-                so_file.copy_to_local(local_path)
-                local_so_files.append(local_path)
+        for so_file in self.input()["analyses"]["collection"].iter_existing():
```

Comment:
Saw this exact code already somewhere above. Maybe make it a common function?

Reply:
Out of scope for this PR. This will be fixed in a new PR for a rewritten RivetMerge